halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/Front50Profile.java

```diff
@@ -43,17 +44,20 @@ public SpinnakerArtifact getArtifact() {
   @Override
   public ProfileConfig generateFullConfig(ProfileConfig config, DeploymentConfiguration deploymentConfiguration, SpinnakerEndpoints endpoints) {
     PersistentStorage storage = deploymentConfiguration.getPersistentStorage();
-    Account account = accountService.getProviderAccount(deploymentConfiguration.getName(), "google", storage.getAccountName());
+    Account account = accountService.getAnyProviderAccount(deploymentConfiguration.getName(), storage.getAccountName());
     Front50Credentials credentials = new Front50Credentials();
 
-    if (account != null) {
-      credentials.getSpinnaker().setGcs(new Front50Credentials.Spinnaker.GCS(storage, (GoogleAccount) account));
+    if (account == null) {
+      throw new RuntimeException("Validation failure: account name expected in PersistentStorage configuration.");
+    }
+
+    if (account instanceof CommonGoogleAccount) {
+      credentials.getSpinnaker().setGcs(new Front50Credentials.Spinnaker.GCS(storage, (CommonGoogleAccount) account));
       config.setRequiredFiles(dependentFiles(account));
       config.extendConfig(config.getPrimaryConfigFile(), yamlToString(credentials));
       return config;
     } else {
-      throw new HalException(
-          new ConfigProblemBuilder(Severity.FATAL, "No valid account configured for persistent storage.").build());
+      throw new RuntimeException("Validation failure: google or appengine account name expected in PersistentStorage configuration.");
```

Comment:
The err msg in `PersistentStorageService` reads "You must supply either a GCE, or Appengine account to your persistent storage service". Are we referring to a Google account or a GCE account?